### ISSUE TITLE
数据集拆分有问题？

### ISSUE THREAD

**Issue by xiyangxia** [inorilzy/renlian#5]
` train_images, valid_images, train_labels, valid_labels = train_test_split(images, labels, test_size=0.3,
                                                                                  random_state=random.randint(0, 100))
        `
这里的数据及拆分是不是有问题？好像是把所有人的照片一起拆了，没有考虑分类吗？
